### ISSUE TITLE
Xfce4 error: From Cannot open /dev/tty0 (Permission denied) to Cannot open virtual console 2 (Permission denied)

### ISSUE THREAD

**Issue by yutouyes** [dnschneid/crouton#4258]
```
chronos@localhost / $ sudo edit-chroot -all
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20200116214215~master:9017d640
release: xenial
architecture: amd64
xmethod: xorg
targets: xfce
host: version 12739.105.0 (Official Build) stable-channel rammus 
kernel: Linux localhost 4.19.98-brunch-sebanc #1 SMP PREEMPT Mon Apr 13 12:03:17 CEST 2020 x86_64 x86_64 x86_64 GNU/Linux
freon: no
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
```

After a fresh install,I ran ```sudo crouton -t xfce -u``` with nothing modified. 
Then tried ```chronos@localhost / $ sudo startxfce4``` , error came with 
```
(EE) 
Fatal server error:
(EE) parse_vt_settings: Cannot open /dev/tty0 (Permission denied)
(EE) 
(EE) 
``` 
According to #3320 I entered 
```
chronos@localhost / $ sudo enter-chroot
Entering /mnt/stateful_partition/crouton/chroots/xenial...
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.
(xenial)yucr@localhost:~$ sudo chmod 666 /dev/tty0
[sudo] password for yucr: 
(xenial)yucr@localhost:~$ ls /dev/tty0 -lah
crw-rw-rw-. 1 root tty 4, 0 Apr 14 11:46 /dev/tty0
sudo usermod -a -G tty  yucr
```

/etc/X11/Xwrapper.config is set as allowed_users=anybody, and packages ```xserver-xorg-legacy``` ```xserver-xorg-legacy-hwe-16.04``` are not installed, so I have nothing to purge.

Then ran startxfce4, another error here:
```
(EE) 
Fatal server error:
(EE) xf86OpenConsole: Cannot open virtual console 2 (Permission denied)
(EE) 
(EE) 
```
I'm still unable to launch xfce4.
I've tried bionic and sid, all of them dropped into the same error.

**Comment by DennisLfromGA** [comment]
@yutouyes 

If you noticed in comment https://github.com/dnschneid/crouton/issues/3320#issuecomment-315075960, the command `sudo chmod 666 /dev/tty0` was *not* the solution.  

If you follow the trail from https://github.com/dnschneid/crouton/issues/3320#issuecomment-315089858 to https://github.com/dnschneid/crouton/issues/3289#issuecomment-315092226 to https://github.com/dnschneid/crouton/issues/3310#issuecomment-313702705 you *may* find a workaround.

*Hope this helps,*
-**DennisLfromGA**

**Comment by yutouyes** [comment]
Thank you @DennisLfromGA 

I've tried #3310 but as I mentioned above, there are no such packages 'xserver-xorg-legacy' 'xserver-xorg-legacy-hwe-16.04'. Although I tried to install them and purge them, nothing happened. At this time the error was still '/dev/tty0(permission denied)', and after 'sudo chmod' it went to 'virtual console 2 (Permission denied)' . If chmod doesn't play a role I can ignore it, but if so, I cannot find a way to pass the first error about '/dev/tty0'

It may be related to the project Brunch, which helping me install CrOS https://github.com/sebanc/brunch/issues/46
Because I just tried crouton -t xfce in another system, Fydeos, and everything perfect. With default configuration, X displayed successfully. 

I've asked the author of Brunch a favor, and wish talents in crouton community can also have some ideas.
